Description: Removes auto generation of version.py by astropy helpers
Author: Josue Ortega <josue@debian.org>
Last-Update: 2016-08-13

--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,7 @@
 from astropy_helpers.setup_helpers import (
     register_commands, adjust_compiler, get_debug_option, get_package_info)
 from astropy_helpers.git_helpers import get_git_devstr
-from astropy_helpers.version_helpers import generate_version_py
+#from astropy_helpers.version_helpers import generate_version_py
 
 # Get some values from the setup.cfg
 try:
@@ -80,8 +80,8 @@
 adjust_compiler(PACKAGENAME)
 
 # Freeze build information in version.py
-generate_version_py(PACKAGENAME, VERSION, RELEASE,
-                    get_debug_option(PACKAGENAME))
+#generate_version_py(PACKAGENAME, VERSION, RELEASE,
+#                    get_debug_option(PACKAGENAME))
 
 # Treat everything in scripts except README.rst as a script to be installed
 scripts = [fname for fname in glob.glob(os.path.join('scripts', '*'))
